mount-daemon/controllers/clientmount_controller.go

```diff
@@ -44,9 +44,10 @@ import (
 // ClientMountReconciler reconciles a ClientMount object
 type ClientMountReconciler struct {
 	client.Client
-	Mock   bool
-	Log    logr.Logger
-	Scheme *runtime.Scheme
+	Mock    bool
+	Timeout uint
```

Comment:
Please always include units (`TimeoutInSeconds`) or convert this to a `time.Duration` and do the conversion in main.go

Reply:
Better yet, checkout `flag.Duration()`